test/integration/retryable-writes/retryable_writes.spec.test.ts

```diff
@@ -0,0 +1,209 @@
+import { expect } from 'chai';
+
+import type { Collection, Db, MongoClient } from '../../../src';
+import { loadSpecTests } from '../../spec';
+import { legacyRunOnToRunOnRequirement } from '../../tools/spec-runner';
+import { isAnyRequirementSatisfied } from '../../tools/unified-spec-runner/unified-utils';
+
+interface RetryableWriteTestContext {
+  client?: MongoClient;
+  db?: Db;
+  collection?: Collection;
+  failPointName?: any;
+}
+
+describe('Legacy Retryable Writes Specs', function () {
```

Comment:
The ticket NODE-3769 mentions that skipped tests should be unskipped for this. Do you know what was being referenced there? I wasn't able to find anything in the code base that was tagged to this ticket or any skipped retryable writes tests for that matter.

Reply:
@durran Do you have context on the tests you mentioned? I didn't come across skipping logic

Reply:
I think I just added that in case tests were skipped but I don't see anything currently in retryable writes actually getting skipped.